airflow/api_fastapi/openapi/v1-generated.yaml

```diff
@@ -123,13 +123,56 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/HTTPValidationError'
+  /public/dags/{dag_id}:
+    patch:
+      tags:
+      - DAG
+      summary: Patch Dag
+      description: Update the specific DAG.
+      operationId: patch_dag_public_dags__dag_id__patch
+      parameters:
+      - name: dag_id
+        in: path
+        required: true
+        schema:
+          type: string
+          title: Dag Id
+      - name: update_mask
+        in: query
+        required: false
+        schema:
+          anyOf:
+          - type: array
+            items:
+              type: string
+          - type: 'null'
+          title: Update Mask
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/DAGPatchBody'
+      responses:
+        '200':
+          description: Successful Response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DAGResponse'
+        '422':
```

Comment:
The connexion API defines HTTP 401, 403 and 404 as response - is it possible to add this to the specs here as well? How are the error codes generated?

Reply:
Good point. I'll do that a follow up PR right after this one because fastapi does not handle this natively and i'll have to indroduce some exception wrapper here and there. Just to keep things separated.

I'll merge this one and ping you on the 'document http error in the swagger for fastapi' one. :)